charts/stable/dnscrypt-proxy/templates/common.yaml

```diff
@@ -0,0 +1,23 @@
+{{/* Make sure all variables are set properly */}}
+{{- include "common.values.setup" . }}
+
+{{/* Append the configMap volume to the volumes */}}
+{{- define "dnscryptproxy.configVolume" -}}
+enabled: "true"
+mountPath: "/config/dnscrypt-proxy.toml"
+subPath: "dnscrypt-proxy.toml"
+type: "custom"
+volumeSpec:
+  configMap:
+    name: {{ include "common.names.fullname" . }}
+{{- end -}}
+
```

Comment:
```suggestion
```

Reply:
Oh... Yes, it's obvious !

I quickly correct it and push again